### PR TITLE
Add support for ScheduledQueryRun

### PR DESCRIPTION
**Pull request by remi-stripe**
r? @brandur-stripe 
cc @stripe/api-libraries

### CODE REVIEW

File: sigma/scheduledqueryrun/client_test.go [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import (
+	"testing"
+
+	assert "github.com/stretchr/testify/require"
+	stripe "github.com/stripe/stripe-go"
+	_ "github.com/stripe/stripe-go/testing"
+)
+
+func TestSigmaScheduledQueryRunGet(t *testing.T) {
+	t.Skip("skipping test as stripe-mock does not support this resource")
```
Comment (remi-stripe) [inline sigma/scheduledqueryrun/client_test.go]: It would be really cool if we could add some way to stub requests like we do in other libraries!

Comment (brandur-stripe) [reply sigma/scheduledqueryrun/client_test.go]: Yes, that would be cool! Mocking isn't really possible in Go, but I think we can accomplish something similar with a helper to boot up a little JSON test server that responds with a blob you can hand it.

In this case though, I think we should make sure that stripe-mock supports scheduled query runs. I'm just winding that through the backend now.

Comment (brandur-stripe) [reply sigma/scheduledqueryrun/client_test.go]: Okay, can you try bumping stripe-mock to 0.24.0 and seeing if you can activate these tests? Thanks!

ptal @remi-stripe

Comment (remi-stripe) [reply sigma/scheduledqueryrun/client_test.go]: oh neat, thanks for shipping this one!

**Comment by remi-stripe** [comment]
@brandur-stripe fixed, PTAL!

**Comment by brandur-stripe** [comment]
Released as 37.0.0.

**Pull request merged**